pliops/producer.cc

```diff
@@ -113,7 +115,9 @@ void Producer::ReaderThread(uint32_t iterator_parallelism_factor, uint32_t threa
   logger->Log(Severity::INFO, FormatString("Reader thread #%d ended. Performed %lld operations.\n", thread_id, total_number_of_operations));
 
   // Release the iterator
+#ifdef XDPROCKS
   status = iterator->Close();
+#endif
```

Comment:
why do only Xdprocks need to call iterator->Close()?

Reply:
this API only exists in XdpRocks